### PR TITLE
Change to use express server

### PR DESCRIPTION
**Pull request by zhouyx**

### CODE REVIEW

File: public/bundle1.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ function addBundle(path) {
 }
 
 console.log(',bundle1-ask-bundle2,', performance.now());
-addBundle('https://localhost:3000/bundle2.js');
+addBundle('https://localhost:8000/bundle2.js');
```
Comment (dvoytenko) [inline public/bundle1.js:14]: Cool. I'll probably redo to use URL resolution.

File: src/server.js [hunk 2]
```diff
@@ -55,38 +34,43 @@ function link(res, path, rel, forPath, headers) {
...
   // Push or link if needed.
-  const headers = {};
-  if (reqPath.endsWith('.js')) {
-    headers['Cache-Control'] = 'public, max-age=600';
-  }
+  const headers = {
+    'content-length': stat.size,
+    'last-modified': stat.mtime.toUTCString(),
+    'Cache-Control': 'public, max-age=600',
+    'Content-Type': 'application/javascript',
```
Comment (dvoytenko) [inline src/server.js:66]: What if it's not a JS? The html files are here as well.

**Review by dvoytenko — CHANGES_REQUESTED** [review]

File: src/server.js (hunk 2 above)
Comment (zhouyx) [reply src/server.js:66]: The middleware checks for 
``
app.get('*.js/', (req, res) => {
  onRequest(req, res);
});
```
It only applies to JS file

Comment (dvoytenko) [reply src/server.js:66]: I see. OK. It works then. I wanted to add some additional massaging for html files, but we can go w/o for now.

**Review by dvoytenko — APPROVED** [review]

File: src/server.js (hunk 2 above)
Comment (zhouyx) [reply src/server.js:66]: Got it. we can easily support that, `express.static()` allows us to setHeader to files easily.

**Pull request merged**